console/.eslintrc

```diff
@@ -3,8 +3,7 @@
   "env": {
     "browser": true,
     "node": true,
-    "mocha": true,
-    "cypress/globals": true
```

Comment:
cypress directory contains it's own `.eslintrc` so this is not required in `console` level